### PR TITLE
Update AppDiscovery.md

### PR DESCRIPTION
**Pull request by andykog**
Fix broken link

### CODE REVIEW

File: Documentation/AppDiscovery.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ This dialog displays a list of applications which can be profiled by Detox Instr
 
 If your app does not appear in the app selection dialog, the following tips will help you troubleshoot.
 
-- Make sure you have integrated your app with the Profiler framework. See [Profiler Framework Integration Guide](Documentation/XcodeIntegrationGuide.md) for more information.
+- Make sure you have integrated your app with the Profiler framework. See [Profiler Framework Integration Guide](/Documentation/XcodeIntegrationGuide.md) for more information.
```
Comment (LeoNatan) [inline Documentation/AppDiscovery.md]: Oops. Thanks!

File: Documentation/AppDiscovery.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ This dialog displays a list of applications which can be profiled by Detox Instr
...
 - Devices should be connected on the same network (Wi-Fi, LAN for macOS & Simulators or Lightning cable for direct connection)
 - Security software, such as Antivirus and Firewall applications, may interfere with Bonjour; please consult the software manual or contact your system administrator
 - Corporate networks may block or otherwise prevent Bonjour from working; please contact your system administrator
-  - As a workaround, you can connect your device locally to your Mac using a Lightning cable, which will create a Bonjour-supported direct connection.
\ No newline at end of file
+  - As a workaround, you can connect your device locally to your Mac using a Lightning cable, which will create a Bonjour-supported direct connection.
```
Comment (LeoNatan) [inline Documentation/AppDiscovery.md]: What has changed here?

Comment (andykog) [reply Documentation/AppDiscovery.md]: Probably, line ending. I've edited this file with Github interface.

Comment (LeoNatan) [reply Documentation/AppDiscovery.md]: 👍

**Pull request merged**